ui/cypress/integration/admin.test.ts

```diff
@@ -0,0 +1,259 @@
+describe('Use Admin tab', () => {
+  let url: string
+  let sourceId: string
+
+  beforeEach(() => {
+    cy.OAuthLogin('test').then(() => {
+      cy.visit('/login')
+      cy.removeConnections()
+      cy.createConnection()
+      cy.get('@connections').then(source => {
+        sourceId = source[0].id
+      })
+    })
+  })
+
+  //   describe('Chronograf', () => {
+  //     beforeEach(() => {
+  //       cy.fixture('routes').then(({adminChronograf}) => {
+  //         url = `/sources/${sourceId}${adminChronograf}`
+  //       })
+  //     })
+
+  //     /* ADMIN TAB CHRONOGRAF */
+  //   })
+
+  describe.only('InfluxDB', () => {
```

Comment:
I think that the commented code is essentially useless herein and could be deleted. Also `describe.only` should be IMHO is herein probably for no reason, or is there is a need to have it? 

```suggestion
  describe('InfluxDB', () => {
```

Reply:
As it stands now, there is no reason to use `describe.only`.  Changed it to describe and removed commented code from the entire file.